app/views/developer.scala.html

```diff
@@ -129,6 +129,9 @@ <h3>Access Attributes</h3>
                             <a id="attributes-link-CSV" target="_blank" href=''>
                                 <code id="attributes-code-CSV"></code>
                             </a>
+                            <a id="attributes-link-ShapeFile" target="_blank" href=''>
```

Comment:
Can you change the id's in all the HTML you added to fully use lowercase in the kebab-case.